Incubator - first draft (#66)
OBF/obf-docs: OBF-Incubator-policy.md

# OBF Incubator Policy
_Status: **Unratified Draft**. This policy is currently being developed. It has not yet been discussed beyond the Board of Directors._

## Background

Occasionally organisations at collaboration festivals, hackathons, and other events may wish to create an open source code, data, policy, document, or design repository, and may wish for a neutral location to store this work. The GitHub organisation OBF-incubator exists in order to provide a low-barrier parent organisation in cases like this.

## Policies

### Creating repositories

During an OBF-organised event (e.g. BOSC and post-BOSC CollaborationFests), people interested in creating a repository under the OBF-Incubator organisation should approach an OBF board member or member of the OBF-Incubator/admins team to request the creation of:

1. A GitHub team for the members of the project
2. A GitHub repository for the desired project. The team in (1) above will have maintainer access to this repository.

### Prerequisites
The requirements for creating repositories such as the above are generally that:

1. The repository must have an OSI-approved licence.

> 2. The repository should open science in some way, although this can be interpreted broadly as needed by the OBF-Incubator admin who is creating the repository.

Reviewer: ```suggestion
2. The repository should promote open science, consistent with the [OBF mission](https://www.open-bio.org/about/).
```

### Maintenance

- The OBF is _not_ responsible for maintenance or support for these repositories, and does not accept any kind of liability for them.
- In cases where the original creators are unable to, or do not wish to support the repository, the OBF reserves the right to archive the repository on GitHub.

### OBF affiliated projects eligibility

Any projects which wish to apply for OBF affiliated project status is welcome to do so, but they will need to go through the same processes as any other organisation. Participating in the incubator does not confer any special statuses within the OBF.

## Organising your own event

> If you wish to organise your own open life science event, and would like to use the OBF-incubator organisation as part of this, please email board@open-bio.org to apply. Approval is at the discretion of the OBF board, and approved applicants will be added to the OBF-Incubator/admins team, enabling them to create repositories for their event attendees.

Reviewer: ```suggestion
If you wish to organise your own open life science event, and would like to use the OBF-incubator organisation as part of this, please email board@open-bio.org to apply. Approval is at the discretion of the OBF board. Approved applicants will be able to create repositories for their event attendees.
```
